### PR TITLE
Update 03-operations-on-variables.mq4

### PR DESCRIPTION
**Pull request by preetihada**
Add code for substractions

### CODE REVIEW

File: 01-programming-core-mql4/03-operations-on-variables.mq4 [hunk 1]
```diff
@@ -21,6 +21,15 @@ void OnStart()
...
    Alert(z);               //should give 23 as output
    
+   //Substraction Operation
+   
+   int x = 15;
+   int y = 8;
+   int z;
+   
+   z = x - y;
+   Alert(z);               //should give 23 as output
```
Comment (rahuldhangar) [inline 01-programming-core-mql4/03-operations-on-variables.mq4:31]: Change the comment in subtraction. 15 - 8 shouldn't give 23 as output.

**Review by rahuldhangar — CHANGES_REQUESTED** [review]
Change the comment in subtraction. 15 - 8 shouldn't give 23 as output.

File: 01-programming-core-mql4/03-operations-on-variables.mq4 (hunk 1 above)
Comment (preetihada) [reply 01-programming-core-mql4/03-operations-on-variables.mq4:31]: yes , it should be 7

**Comment by rahuldhangar** [comment]
This change is merged in PR #5 Closing this one as its conflicting and obsolete.

**Review by rahuldhangar — COMMENTED** [review]
Implemented by you in PR #5 as discussed. Closing this Review.

**Pull request closed without merging**